letsencrypt-auto-source/tests/oraclelinux6_tests.sh

```diff
@@ -27,55 +24,31 @@ if [ $RESULT -ne 0 ]; then
   exit 1
 fi
 
-echo "PASSED: bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
+echo "PASSED: Bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
 
-# Expect letsencrypt-auto to just fail to rebootstrap and start certbot in interactive
-# shell since SCL is not installed.
-if ! "$LE_AUTO" --version 2>&1 | grep -q "Enable the SCL repository and try running Certbot again."; then
+# Expect letsencrypt-auto to skip rebootstrapping with a warning since SCL is not installed.
+if ! "$LE_AUTO" --non-interactive --version 2>&1 | grep -q "Skipping upgrade because new OS dependencies may need to be installed."; then
   echo "FAILED: Script letsencrypt-auto managed to start Certbot in interactive shell while SCL is not enabled!"
   exit 1
 fi
 
-echo "PASSED: Script letsencrypt-auto did not rebootstrap and did not star Certbot."
-
-# Expect letsencrypt-auto to not update certbot, but start it anyway in non-interactive
-# shell since SCL is not installed.
-# NB: Readline has an issue on all Python versions for CentOS 6, making `certbot --version`
-# output an unprintable ASCII character on a new line at the end.
-# So we take the second last line of the output.
-version=$($LE_AUTO --version 2>/dev/null | tail -2 | head -1)
-
-if [ -z "$version" ]; then
-  echo "ERROR: Script letsencrypt-auto failed to start certbot in a non-interactive shell while SCL was not enabled."
-  exit 1
-fi
-
-if [ "$version" != "$INITIAL_CERTBOT_VERSION_PY34" ]; then
```

Comment:
Why delete this check? I'd personally like to keep it to ensure we don't print a warning and upgrade Certbot anyway.

Reply:
Added back the test.